tests/test_dashboard_page.py

```diff
@@ -25,16 +27,32 @@ def test_footer_all_elements_visible_and_clickable_tc_002(self, locator):
         assert driver.is_visible(locator)
         assert driver.is_clickable(locator)
 
-    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all)
+    def test_menu_selector_can_be_push_tc_005(self):
+        driver = DashboardPage(self.driver)
+        driver.click(DashboardPageLocators.RIGHT_ARROW_SELECTOR)
+        assert driver.is_visible(DashboardPageLocators.RIGHT_ARROW_SELECTOR_ALL_VISIBLE)
+        driver.click(DashboardPageLocators.RIGHT_ARROW_SELECTOR_ALL)
+        assert driver.get_title() == Titles.TITLE_DASHBOARD_PAGE
+        driver.go_to_page(TD.BASE_URL)
+
+    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all,
+                             ids=DashboardPageLocators.ids_dashboard_all)
     def test_dashboard_all_element_is_visible_tc_008(self, locator):
         driver = DashboardPage(self.driver)
         assert driver.is_visible(locator)
 
-    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_text_field)
+    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_text_field,
+                             ids=DashboardPageLocators.ids_dashboard_text_field)
     def test_dashboard_all_element_is_clickable_tc_009(self, locator):
         driver = DashboardPage(self.driver)
         assert driver.is_clickable(locator)
 
+    def test_dashboard_new_item_clickable_tc_010(self):
+        driver = DashboardPage(self.driver)
+        driver.click(DashboardPageLocators.TEXT_NEW_ITEM)
+        assert driver.get_title() == Titles.TITLE_NEW_ITEM
+        driver.go_to_page(TD.BASE_URL)
```

Comment:
It looks like this test is verifying title of a different page, i mean not DashboardPAge.If it is, it belongs to a different test page.

Reply:
I have a problem with driver.current_url - I need help to solve it

Reply:
now added skip this test